### DISCUSSION TITLE
Migration of Sunbird-mobile-app to Android SDK-34

### DISCUSSION THREAD

**Discussion by siddhishinde0723** [Sunbird-Ed/Community#743] [General Discussion]
Hi Team,

Any update on migrating the app to SDK 34, we have only 2 days left for the Google deadline. 

@aishwaryashikshalokam @Prateek-slokam @varsha-sl @MeghnaSL @ashokreddy1208 @kameshbhr @mmanimegalai @rajeevsathish @chitranshu-keshav @anisha147 @vinukumar-vs @Krishnaj20 @kiranharidas187 @swayangjit

**Comment by HarishGangula** [comment]
@siddhishinde0723 You can update it with configuration value of  target sdk

**Reply by siddhishinde0723** [reply]
@HarishGangula Only this thing needs to be changed? plugins and other dependencies will remain the same?
` defaultBuildToolsVersion="34.0.2" //String
      defaultMinSdkVersion=22 //Integer - Minimum requirement is Android 4.4
      defaultTargetSdkVersion=34 //Integer - We ALWAYS target the latest by default
      defaultCompileSdkVersion=34 //Integer - We ALWAYS compile with the latest by default`

cc: @sayali2523

**Reply by HarishGangula** [reply]
@siddhishinde0723  are you facing any issues with the changes it might be because of the plugins dependency version needs to be updated.

**Reply by siddhishinde0723** [reply]
@HarishGangula after updating the configuration value the app is getting crashed.
@sayali2523

**Reply by HarishGangula** [reply]
@siddhishinde0723  Need more info to provide the solution for it.

**Reply by siddhishinde0723** [reply]
@HarishGangula we are trying to update sunbird app to sdk 34 we have just updated the build.gradle file.
Is there anything else we need to update bcause only changing the config the app is getting crashed.

**Reply by HarishGangula** [reply]
@siddhishinde0723  Since it is ionic based app, You follow the guide from ionic framework to do the migration, If you face any issue which are specific to the sunbird functionality or configurations, Share with us to resolve them.

**Comment by HarishGangula** [comment]
As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
